dask_planner/src/sql/types.rs

```diff
@@ -246,6 +246,8 @@ impl SqlTypeName {
             DataType::Float16 => Ok(SqlTypeName::REAL),
             DataType::Float32 => Ok(SqlTypeName::FLOAT),
             DataType::Float64 => Ok(SqlTypeName::DOUBLE),
+            DataType::Time32(_unit) => Ok(SqlTypeName::TIME),
+            DataType::Time64(_unit) => Ok(SqlTypeName::TIME),
```

Comment:
We can combine these into one match:

```suggestion
            DataType::Time32(_) | DataType::Time64(_) => Ok(SqlTypeName::TIME),
```